src/Toolkit/Mime.php

```diff
@@ -239,7 +240,20 @@ public static function isAccepted(string $mime, string $pattern): bool
     }
 
     /**
-     * Returns the extension for a given mime type
+     * Tests if a MIME wildcard pattern from an `Accept` header
+     * matches a given type
+     *
+     * @param string $wildcard
+     * @param string $test
+     * @return bool
+     */
+    public static function matchWildcard(string $wildcard, string $test): bool
+    {
+        return fnmatch($wildcard, $test, FNM_PATHNAME) === true;
+    }
```

Comment:
I'd like to restructure this a bit. 

```php
public static function matches(string $mime, string $wildcard): bool
{
    return fnmatch($wildcard, $mime, FNM_PATHNAME) === true;
}
```

In my opinion, it fits better to the way we normally handle arguments in such static methods. i.e. see `Mime::isAccepted` or something like `Str::contains`. We always pass the "haystack" first and then the "needle" second. 

```
Mime::matches('text/html', 'text/*');
```

Reply:
The rest is great!